tests/unit-tests/internal/emails/test-class-email-list-table.php

```diff
@@ -67,7 +67,7 @@ public function testPrepareItems_WhenPaginated_SetsTheCorrectOffset() {
 					'post_type'      => Email_Post_Type::POST_TYPE,
 					'posts_per_page' => 20,
 					'offset'         => 20,
-					'meta_key'       => 'sensei_email_description', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
+					'meta_key'       => '_sensei_email_description', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
```

Comment:
Should we use the constant reference?

Reply:
In tests definitely no.
That's definite for me :) So I explain. In case we introduced an error in our constant, we want to catch it here. So we don't use the same constant, but we use raw values in tests.